CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
 # Changelog
 
-## 0.5.0
+## Unreleased
```

Comment:
A few items under 0.5.0 now are under `Unreleased`, is that right?

Reply:
Yes, I noticed some unreleased changes ended up on the 0.5.0 changelog  so I fixed it on this PR